.rubocop_todo.yml

```diff
@@ -1,7 +1,101 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2017-09-21 12:15:24 -0400 using RuboCop version 0.47.1.
+# on 2017-11-16 15:45:10 -0500 using RuboCop version 0.50.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 22
+# Cop supports --auto-correct.
+Layout/EmptyLineAfterMagicComment:
+  Enabled: false
+
+# Offense count: 1
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
+# URISchemes: http, https
+Metrics/LineLength:
+  Max: 180
+
+# Offense count: 1
+# Configuration parameters: ExpectMatchingDefinition, Regex, IgnoreExecutableScripts, AllowedAcronyms.
+# AllowedAcronyms: CLI, DSL, ACL, API, ASCII, CPU, CSS, DNS, EOF, GUID, HTML, HTTP, HTTPS, ID, IP, JSON, LHS, QPS, RAM, RHS, RPC, SLA, SMTP, SQL, SSH, TCP, TLS, TTL, UDP, UI, UID, UUID, URI, URL, UTF8, VM, XML, XMPP, XSRF, XSS
+Naming/FileName:
+  Exclude:
+    - 'Guardfile'
+
+# Offense count: 1
+Performance/UnfreezeString:
+  Exclude:
```

Comment:
Why are you setting up all of these excludes (here and below) instead of fixing the files? I'm unclear as to what the point of upgrading rubocop is if the only file you modified as a result of the new rules was Guardfile.

Reply:
I upgraded Rubocop because of a security vulnerability.  I don't have the time right now to fix all the new offenses it's complaining about or turn off the specific cops.